indico/modules/rb_new/client/js/components/TimeRangePicker.module.scss

```diff
@@ -0,0 +1,28 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+.time-dropdown {
+    min-width: 90px !important;
+    margin-right: 14px;
+    text-align: center !important;
+}
+
+.duration {
+    font-size: x-small;
```

Comment:
do we really want to use that instead of a value in `em`?

Reply:
yeah, em seems better